CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.0.0-beta2
+ - Massive internal refactor of client handling
+ - Background HTTP sniffing support
+
+## 2.0.0-beta
+ - Only support HTTP Protocol
```

Comment:
Can we be explicit here to say we removed node and transport protocol code and this has been moved to native plugin?